### PR TITLE
Add Wait parameter description.

### PR DESCRIPTION
**Pull request by velkovb**

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012r2-ps/failoverclusters/Stop-Cluster.md [hunk 1]
```diff
@@ -149,7 +151,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (get-itips) [inline docset/winserver2012r2-ps/failoverclusters/Stop-Cluster.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Comment by velkovb** [comment]
@get-itips I just did all those PRs before I found out about the http to https thing. From now on I will do it on all PRs.

**Comment by get-itips** [comment]
> @get-itips I just did all those PRs before I found out about the http to https thing. From now on I will do it on all PRs.

awesome

**Comment by v-raaba** [comment]
Hi @velkovb  - We understand that these [PR criteria](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master) are new, and we appreciate your patience with the process. 

**Following are blocking issues that need to be resolved before we can merge:**

Conflicting files
docset/windows/failoverclusters/stop-cluster.md 

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)



***

<Use this space to insert any comments.>

Thanks

**Comment by velkovb** [comment]
> Hi @velkovb - We understand that these [PR criteria](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master) are new, and we appreciate your patience with the process.
> 
> **Following are blocking issues that need to be resolved before we can merge:**
> 
> Conflicting files
> docset/windows/failoverclusters/stop-cluster.md
> 
> **Reference**
> 
> [Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)
> 
> Thanks

@v-raaba 
image
I am not able to open the criteria you are referring to.

**Pull request merged**